lib/internal/inspector_network_tracking.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const {
+  DateNow,
+} = primordials;
+
+const {
+  getOptionValue,
+} = require('internal/options');
+
+let dc;
+let NodeNetwork;
+
+let requestId = 0;
+const getNextRequestId = () => `node-network-event-${++requestId}`;
+
+function onClientRequestStart({ request }) {
+  const url = `${request.protocol}//${request.host}${request.path}`;
+  const wallTime = DateNow();
+  const timestamp = wallTime / 1000;
+  request._inspectorRequestId = getNextRequestId();
+  NodeNetwork.requestWillBeSent({
+    requestId: request._inspectorRequestId,
+    timestamp,
+    wallTime,
+    request: {
+      url,
+      method: request.method,
+    },
+  });
+}
+
+function onClientResponseFinish({ request }) {
+  if (typeof request._inspectorRequestId !== 'string') {
+    return;
+  }
+  const timestamp = DateNow() / 1000;
+  NodeNetwork.responseReceived({
+    requestId: request._inspectorRequestId,
+    timestamp,
+  });
+  NodeNetwork.loadingFinished({
+    requestId: request._inspectorRequestId,
+    timestamp,
+  });
+}
+
+function enable() {
+  if (!getOptionValue('--experimental-network-inspection')) {
```

Comment:
This and the similar call below should be hoisted to the `setupNetworkInspection()` in `lib/internal/process/pre_execution.js` if it's unused when not flagged.

Reply:
Done 4e02199e5810b9ffcb3179aeaa0e11b990874d0f